spark-jobs/src/main/scala/za/co/absa/enceladus/standardization_conformance/config/StandardizationConformanceConfig.scala

```diff
@@ -0,0 +1,112 @@
+import za.co.absa.enceladus.standardization.config.StandardizationConfigParser
+
+import scala.util.Try
+
+
+case class StandardizationConformanceConfig(datasetName: String = "",
+                                            datasetVersion: Int = 1,
+                                            reportDate: String = "",
+                                            reportVersion: Option[Int] = None,
+                                            menasCredentialsFactory: MenasCredentialsFactory = InvalidMenasCredentialsFactory,
+                                            performanceMetricsFile: Option[String] = None,
+                                            folderPrefix: Option[String] = None,
+                                            persistStorageLevel: Option[StorageLevel] = None,
+
+                                            publishPathOverride: Option[String] = None,
+                                            experimentalMappingRule: Option[Boolean] = None,
+                                            isCatalystWorkaroundEnabled: Option[Boolean] = None,
+                                            autocleanStandardizedFolder: Option[Boolean] = None,
+
+                                            rawFormat: String = "xml",
+                                            charset: Option[String] = None,
+                                            rowTag: Option[String] = None,
+                                            csvDelimiter: Option[String] = None,
+                                            csvHeader: Option[Boolean] = Some(false),
+                                            csvQuote: Option[String] = None,
+                                            csvEscape: Option[String] = None,
+                                            cobolOptions: Option[CobolOptions] = None,
+                                            fixedWidthTrimValues: Option[Boolean] = Some(false),
+                                            rawPathOverride: Option[String] = None,
+                                            failOnInputNotPerSchema: Boolean = false,
+
+                                            credsFile: Option[String] = None,
+                                            keytabFile: Option[String] = None) extends StandardizationConfigParser[StandardizationConformanceConfig]
+  with ConformanceConfigParser[StandardizationConformanceConfig]{
+
+  override def withPublishPathOverride(value: Option[String]): StandardizationConformanceConfig = copy(publishPathOverride = value)
+  override def withExperimentalMappingRule(value: Option[Boolean]): StandardizationConformanceConfig = copy(experimentalMappingRule = value)
+  override def withIsCatalystWorkaroundEnabled(value: Option[Boolean]): StandardizationConformanceConfig = copy(isCatalystWorkaroundEnabled = value)
+  override def withAutocleanStandardizedFolder(value: Option[Boolean]): StandardizationConformanceConfig = copy(autocleanStandardizedFolder = value)
```

Comment:
Also too long lines, here perhaps worth to mark as exception `//scalastyle:ignore`